cogboard-webapp/src/components/widgets/dialogFields/index.js

```diff
@@ -57,6 +58,25 @@ const dialogFields = {
     step: 1,
     initialValue: 200
   },
+  Text: {
+    component: TextInput,
+    name: 'text',
+    label: 'Text'
+  },
+  TextSize: {
+    component: TextSizeInput,
+    name: 'textSize',
+    label: 'Text Size',
+    dropdownItems: {
```

Comment:
In my opinion this should be harcoded inside `TextSizeInput`

Reply:
We have the same for EndpointInput. Leave it as it is.